imapclient/imapclient.py

```diff
@@ -1858,6 +1858,10 @@ class IMAPlibLoggerAdapter(LoggerAdapter):
     """Adapter preventing IMAP secrets from going to the logging facility."""
 
     def process(self, msg, kwargs):
+        # Fix a case where UTF-8 encoded bytes reach the logger.
+        if isinstance(msg, bytes):
+            msg = msg.decode('utf-8')
```

Comment:
This _only_ works in the specific case of `_send_literal`, which guarantees an ASCII string (i.e. can be decoded as UTF-8).

As per my previous comment (https://github.com/closeio/imapclient/pull/2#discussion_r362979053): if somebody passes `bytes` (or any other object other than `str`) into the logger, you can already assume it's not a command, i.e. does not start with `>` (or it does, but has a different meaning), so you could just omit the `for` loop and go straight to `super`. I believe the `LoggerAdapter` in Python already uses `repr()` so it shouldn't throw an error with a `bytes` object (or any other object).